replay/models/cql.py

```diff
@@ -154,10 +153,8 @@ class CQL(Recommender):
     
     def __init__(
```

Comment:
Could you please move stable variables to class variables? Probably `actor_optim_factory, critic_optim_factory, temp_optim_factory, alpha_optim_factory, actor_encoder_factory, critic_encoder_factory, q_func_factory, n_frames, n_steps, n_critics, initial_temperature, initial_alpha, alpha_threshold, n_action_samples, soft_q_backup, scaler, reward_scaler`

Reply:
Honestly, I am not sure it's a good idea make them class variables since they aren't expected to be read-only for instances, which makes changing them error-prone.